tests/datasets/test_eurosat.py

```diff
@@ -84,14 +60,14 @@ def test_add(self, dataset: EuroSAT) -> None:
         assert len(ds) == 4
 
     def test_already_downloaded(self, dataset: EuroSAT, tmp_path: Path) -> None:
-        EuroSAT(root=tmp_path, download=True)
+        type(dataset)(tmp_path)
```

Comment:
Testing got a bit ugly. We don't actually _have_ to test EuroSATSpatial or EuroSAT100, we could just test EuroSAT and get 100% coverage.